classes/mail/variables/ContextEmailVariable.php

```diff
@@ -109,7 +108,7 @@ protected function getContextUrl(): string
     protected function getContextSignature(array $values): string
     {
         $signature = Mail::compileParams(
-            (string) $this->context->getData('emailSignature'), 
+            (string) $this->context->getData('emailSignature'),
```

Comment:
Shouldn't we also escape email signature with `PKPString::stripUnsafeHtml()`?

Reply:
You're right -- fixed, thanks!